public/app/features/teams/TeamPages.tsx

```diff
@@ -118,49 +107,25 @@ export class TeamPages extends PureComponent<Props, State> {
     return text1.toLocaleLowerCase() === text2.toLocaleLowerCase();
   };
 
-  hideTabsFromNonTeamAdmin = (pageNav: NavModelItem, isSignedInUserTeamAdmin: boolean) => {
-    if (contextSrv.accessControlEnabled()) {
-      return pageNav;
-    }
-
-    if (!isSignedInUserTeamAdmin && pageNav && pageNav.children) {
-      pageNav.children
-        .filter((navItem) => !this.textsAreEqual(navItem.text, PageTypes.Members))
-        .map((navItem) => {
-          navItem.hideFromTabs = true;
-        });
-    }
-
-    return pageNav;
-  };
-
-  renderPage(isSignedInUserTeamAdmin: boolean): React.ReactNode {
+  renderPage(): React.ReactNode {
     const { isSyncEnabled } = this.state;
-    const { members, team } = this.props;
+    const { team } = this.props;
     const currentPage = this.getCurrentPage();
 
-    const canReadTeam = contextSrv.hasAccessInMetadata(
-      AccessControlAction.ActionTeamsRead,
-      team!,
-      isSignedInUserTeamAdmin
-    );
-    const canReadTeamPermissions = contextSrv.hasAccessInMetadata(
+    const canReadTeam = contextSrv.hasPermissionInMetadata(AccessControlAction.ActionTeamsRead, team!);
+    const canReadTeamPermissions = contextSrv.hasPermissionInMetadata(
       AccessControlAction.ActionTeamsPermissionsRead,
-      team!,
-      isSignedInUserTeamAdmin
+      team!
     );
-    const canWriteTeamPermissions = contextSrv.hasAccessInMetadata(
+    const canWriteTeamPermissions = contextSrv.hasPermissionInMetadata(
       AccessControlAction.ActionTeamsPermissionsWrite,
-      team!,
-      isSignedInUserTeamAdmin
+      team!
     );
 
     switch (currentPage) {
       case PageTypes.Members:
-        if (contextSrv.accessControlEnabled()) {
```

Comment:
Is the `accessControl` system available everywhere now days?

Reply:
Yup, starting from Grafana 10 we have removed the option to disable role-based access control. So it is always enabled for everyone.